src/module/storeData.js

```diff
@@ -0,0 +1,16 @@
+// export class UI {
+//   constructor(Data) {
+//     this.newData = Data;
+//   }
+
+//   pushData (dataObj) {
+//     this.newData.push(dataObj);
+//     localStorage.setItem('data', JSON.stringify(this.newData));
+//   }
+
+//   readData() {
+//     const newData = JSON.parse(localStorage.getItem('data') || '[]');
+//     console.log(newData);
+//     this.pushData()
+//   }
+// }
```

Comment:
- [ ] [REQUIRED] Please delete `unnecessary` comments from your codebase so your codebase will be cleaner 👍🏼